platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/hash/VirtualHashListener.java

```diff
@@ -25,14 +25,17 @@
  * Listens to various events that occur during the hashing process.
  */
 public interface VirtualHashListener<K extends VirtualKey, V extends VirtualValue> {
+
     /**
      * Called when starting a new fresh hash operation.
+     *
+     * @param reconnectFlushInterval the number of nodes to hash before they are flushed to disk
      */
-    default void onHashingStarted() {}
+    default void onHashingStarted(int reconnectFlushInterval) {}
```

Comment:
It doesn't look right. `reconnectFlushInterval` (which btw should be renamed to just `flushInterval`) is specific to `AbstractHashListener`. Other `VirtualHashListener` implementations don't need it. So I suggest to leave this method unchanged, and pass the flush interval value to `AbstractHashListener` constructor

Reply:
I see. I went with this path originally, as I wanted to make as less changes as possible and in similar way as it was done already (`reconnectFlushInterval` was initialized in `onHashingStarted`). 
If we go with adding `reconnectFlushInterval` into the `AbstractHashListener`, then it should be added in other `VirtualHashListener` implementations constructors as `AbstractHashListener` is only used there (as `super` call).
Let's quickly sync on it.

Reply:
`AbstractHashListener` has only two subclasses: `ReconnectHashListener` and `FullLeafRehashHashListener`. Only three constructors need to be changed